utils/pack_content.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+#
+# A helper script to count content in a pack.
+#
+import os
```

Comment:
I think we should depend on st2 (simply clone it, add components to PYTHONPATH) and re-use existing functionality from st2common.

In fact, that's what I need to do anyway for the validation changes (clone st2 and add it to PYTHONPATH for  the validation script).